src/insns/cbuildcap_32bit.adoc

```diff
@@ -24,8 +24,12 @@ sealed, `cs1` 's permissions and bounds are equal or a superset of `cs2` 's,
 `cs2` 's bounds are not malformed (see
 xref:section_cap_malformed[xrefstyle=short]), and all reserved bits in `cs2` 's
 metadata are 0. <<CBUILDCAP>> is typically used alongside <<CSETHIGH>> to build
-capabilities from integer values. The case `cs1 == 0` trivially leads to the tag
-of `cd` being stripped.
+capabilities from integer values.
+
+NOTE: The case `cs1 == 0` trivially leads to the tag of `cd` being stripped.
+We opted to not make this a pseudo instruction to clear the tag of capability.
+`cbuildcap cd, c0, cs2` will behave differently in the case of otypes being
+added. Thus we reserve the encoding `cs1 == 0` for now.
```

Comment:
Thanks for the change! Just a couple of notes:
* Please avoid writing sentences in the first person (i.e. I, we, etc). It is not common to do this in specifications.
* It would be good to remove mention to "otype" since this is not defined anywhere in this spec. I would suggest instead writing "future extensions" or something like that.

Reply:
NOTE: Although currently this will set the tag to 0 and leave the metadata otherwise unchanged when `cs1` is `c0`, this may change in future extensions, and so software should not assume this.

perhaps?

Note it's not really a reserved encoding; it doesn't trap and it's guaranteed to do exactly what you asked it to do.

Reply:
I would be happy with that note @jrtc27, but I would change the "should not" to "must not"

Reply:
It is legitimate for software to assume it if it can guarantee (or require) the lack of such extensions, e.g. in an embedded environment where you know the exact targeted hardware